wiki/People/Staff_Log/en.md

```diff
@@ -973,12 +973,40 @@ All users listed below were added on 2017-12-11.
 -   Removed [Wafu](/users/888955) from the Beatmap Nominators due to resignation (2018-02-25)
 -   Removed [Voli](/users/2522275) from the Beatmap Nominators due to resignation (2018-02-28)
 
-#### NewBN additions
+#### New BN additions
 
 -   Added [Kuron-Kun](/users/2697284) to the Beatmap Nominators (2018-02-06)
 -   Added [ailv](/users/6933054) to the Beatmap Nominators (2018-02-26)
 
 ### March
 
 -   Added [Lasse](/users/896613) to the Quality Assurance Team (2018-03-01)
--   Moved [Stefan](/users/626907) to the Global Moderation Team (2018-03-11)
\ No newline at end of file
+-   Moved [Stefan](/users/626907) to the Global Moderation Team (2018-03-11)
+-   Moved [Okoratu](/users/1623405) to the osu! Alumni and Beatmap Nominators (2018-03-18)
+-   Moved [shARPII](/users/776257) to the osu! Alumni  
+-   Added [-Mo-](/users/2202163) to the Quality Assurance Team (2018-03-22)
+-   Added [Gabe](/users/654108) to the Quality Assurance Team (2018-03-22)
+-   Added [Lanturn](/users/1446665) to the Quality Assurance Team (2018-03-22)
+-   Added [Mir](/users/8688812) to the Quality Assurance Team (2018-03-22)
+
+#### BN removals
+
+-   Removed [Naotoshi](/users/5364763) from the Beatmap Nominators due to resignation (2018-03-11)
```

Comment:
`from the Beatmap Nominators` seems repetitive since the title says it all. Idk about `Removed` because it won't be a sentence with it gone.